guides/plugins/plugins/storefront/add-listing-filters.md

```diff
@@ -0,0 +1,174 @@
+# Add custom listing filters
+
+## Overview
+
+In an online shop, filters are an important feature. So you might use filters in your custom plugin. This guide will
+get you covered on how to implement your own, custom filters in Shopware's storefront.
+
+## Prerequisites
+
+Before you starting reading this guide, make sure you got an own plugin installed to work with. If you need a starting 
+point for that, see this guide:
+
+{% page-ref page="./../plugin-base-guide.md" %}
+
+## Create new Filter
+
+At first, you need to create a subscriber -  in this example we'll call it `ExampleListingSubscriber`. If you're not
+sure on working with subscribers, please refer to the guide on working with events in Shopware:
+
+{% page-ref page="./../plugin-fundamentals/listening-to-events.md" %}
+
+As you're familiar with other customization, we'll start creating this new class in the same path as you're seeing in
```

Comment:
"As you're familiar with other customization"

What do you want to say here ? That a plugin developer usually should copy Shopware's business logic directory structure ? If so, I find this sentence highly confusing. I'd suggest sth easier like:

```suggestion
As usual, we'll start by creating this new class in the same path as you're seeing in
```